core/src/main/java/com/github/mizool/core/concurrent/SynchronizerApi.java

```diff
@@ -0,0 +1,163 @@
+package com.github.mizool.core.concurrent;
```

Comment:
Note to self: add license boilerplate

Reply:
@lesmou and I have already scheduled a kickoff for May 21 on this (yes, that far into the future, thanks to various vacations).

Reply:
I don't care about the far-ness into the future-ness. But I'm done with "yeah, let's do this some time soon" and then not doing it.

A scheduled meeting is already an improvement as it puts meaning behind the idea, regardless of when it will happen.